src/frontend/test/validation.js

```diff
@@ -159,7 +159,9 @@ const validateRequiredFields = (form) => {
 
 const validateRequiredFieldsetCheckboxes = (fieldset) => {
   let isValid = false;
-  fieldset.find("input[type=checkbox]").each((i, field) => {
+  const checkboxes = fieldset.find("input[type=checkbox]");
+  if(checkboxes.length === 0) return true;
+  checkboxes.each((i, field) => {
```

Comment:
Not sure if the above is intentional?

Reply:
I'll be honest, we've hit "that point" we were discussing previously about having the changes to the files in 2 places at the same time - it's becoming difficult to keep track of what's where. I've reverted this one, and it needs to be kept "as is". :smile: